tuf/utils/utils.go

```diff
@@ -146,3 +147,15 @@ func FindRoleIndex(rs []*data.Role, name string) int {
 	}
 	return -1
 }
+
+// URLFilePath generates the appropriate HTTP URL path for the targets file,
```

Comment:
Yep, sorry, updated filename because it specifically referenced targets but missed the comment :-/